lighthouse-core/config/config.js

```diff
@@ -145,6 +145,30 @@ function validatePasses(passes, audits, rootPath) {
   });
 }
 
+function validateCategories(categories, audits) {
+  if (!categories || !audits) {
+    return;
+  }
+
+  const auditIds = new Set(audits.map(audit => audit.meta.name));
+  Object.keys(categories).forEach(categoryId => {
+    const category = categories[categoryId];
+    if (!category.audits) {
+      throw new Error(`${categoryId} category has no audits`);
```

Comment:
do we care about this case? Seems more like a `log.warn` situation, maybe? Also this throws an error for no `audits` array, but not for an empty array (which would be covered by "has no audits")